pkg/serverless/tags/tags.go

```diff
@@ -229,10 +230,20 @@ func getRuntimeFromOsReleaseFile(osReleasePath string) string {
 	return runtime
 }
 
-func getRuntime(procPath string, osReleasePath string, varName string) string {
-	foundRuntimes := proc.SearchProcsForEnvVariable(procPath, varName)
-	runtime := cleanRuntimes(foundRuntimes)
-	runtime = strings.Replace(runtime, "AWS_Lambda_", "", 1)
+func getRuntime(procPath string, osReleasePath string, varName string, retries int) string {
+	runtime := ""
+	counter := 0
+	start := time.Now()
+	for len(runtime) == 0 && counter <= retries {
+		if counter > 0 {
+			time.Sleep(5 * time.Millisecond)
+		}
+		foundRuntimes := proc.SearchProcsForEnvVariable(procPath, varName)
+		runtime = cleanRuntimes(foundRuntimes)
+		runtime = strings.Replace(runtime, "AWS_Lambda_", "", 1)
+		counter++
+	}
```

Comment:
Retry searching for the runtime env var up to 5 times, waiting 5ms in between each try.

The total possible latency we could add is: 50ms + time taken for each proc/env var search. This comes out to about ~53ms, as the env var search itself is quick. 

Between node/py/csharp/java, runtimes other than csharp succeed on the first try and find the runtime in < 2ms.

Csharp takes 4 tries, finding the runtime in ~18ms.